schema.graphql

```diff
@@ -564,3 +564,12 @@ type Liquidation @entity {
   fCashMaturities: [BigInt!]
   fCashNotionalTransfer: [BigInt!]
 }
+
+# Data accumulated and condensed into day stats for lending and borrowing
+type LendBorrowDayData @entity {
+  id: ID!
+  date: Int!
```

Comment:
You can add a references to Currency, Market, and associated Trade objects here by:
```
currency: Currency!
market: Market!
trade: [Trade!]!
```

Storing the trade array might be helpful if you eventually want to list out all the trades in a day or something.